CHANGELOG.md

```diff
@@ -1,5 +1,315 @@
+        <a class="nhsuk-back-to-top__link" href="#nhsuk-nav-a-z">
+          <svg class="nhsuk-icon nhsuk-icon__arrow-right" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" aria-hidden="true" width="19" height="19">
+            <path d="M19.6 11.66l-2.73-3A.51.51 0 0 0 16 9v2H5a1 1 0 0 0 0 2h11v2a.5.5 0 0 0 .32.46.39.39 0 0 0 .18 0 .52.52 0 0 0 .37-.16l2.73-3a.5.5 0 0 0 0-.64z"></path>
+          </svg>
+          Back to top
+        </a>
+      </div>
+    </div>
+  </li>
+  <li>
+    <div class="nhsuk-list-panel">
+      <h2 class="nhsuk-list-panel__label" id="C">C</h2>
+      <ul class="nhsuk-list-panel__list nhsuk-list-panel__list--with-label">
+        <li class="nhsuk-list-panel__item">
+          <a class="nhsuk-list-panel__link" href="/conditions/chest-pain/">Chest pain</a>
+        </li>
+        <li class="nhsuk-list-panel__item">
+          <a class="nhsuk-list-panel__link" href="/conditions/cold-sores/">Cold sore</a>
+        </li>
+      </ul>
+      <div class="nhsuk-back-to-top">
+        <a class="nhsuk-back-to-top__link" href="#nhsuk-nav-a-z">
+          <svg class="nhsuk-icon nhsuk-icon__arrow-right" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" aria-hidden="true" width="19" height="19">
+            <path d="M19.6 11.66l-2.73-3A.51.51 0 0 0 16 9v2H5a1 1 0 0 0 0 2h11v2a.5.5 0 0 0 .32.46.39.39 0 0 0 .18 0 .52.52 0 0 0 .37-.16l2.73-3a.5.5 0 0 0 0-.64z"></path>
+          </svg>
+          Back to top
+        </a>
+      </div>
+    </div>
+  </li>
+  <li>
+    <div class="nhsuk-list-panel">
+      <h2 class="nhsuk-list-panel__label" id="D">D</h2>
+      <ul class="nhsuk-list-panel__list nhsuk-list-panel__list--with-label">
+        <li class="nhsuk-list-panel__item">
+          <a class="nhsuk-list-panel__link" href="/conditions/dandruff/">Dandruff</a>
+        </li>
+        <li class="nhsuk-list-panel__item">
+          <a class="nhsuk-list-panel__link" href="/conditions/dementia/">Dementia</a>
+        </li>
+        <li class="nhsuk-list-panel__item">
+          <a class="nhsuk-list-panel__link" href="/conditions/toothache/">Dental pain</a>
+        </li>
+      </ul>
+      <div class="nhsuk-back-to-top">
+        <a class="nhsuk-back-to-top__link" href="#nhsuk-nav-a-z">
+          <svg class="nhsuk-icon nhsuk-icon__arrow-right" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" aria-hidden="true" width="19" height="19">
+            <path d="M19.6 11.66l-2.73-3A.51.51 0 0 0 16 9v2H5a1 1 0 0 0 0 2h11v2a.5.5 0 0 0 .32.46.39.39 0 0 0 .18 0 .52.52 0 0 0 .37-.16l2.73-3a.5.5 0 0 0 0-.64z"></path>
+          </svg>
+          Back to top
+        </a>
+      </div>
+    </div>
+  </li>
+</ol>
+```
+
+The list panel component can be recreated by adding `nhsuk-list--border` to the [list styles in typography](https://service-manual.nhs.uk/design-system/styles/typography#lists)
+
+#### New code:
+
+```
+<ul class="nhsuk-list nhsuk-list--border">
+          <li><a href="#">List item as a link</a></li>
```

Comment:
Can the spacing/tabbing be fixed here? ;-)

Reply:
fixed 😁